lib/src/api/converters.dart

```diff
@@ -49,6 +49,37 @@ class NullableDateTimeConverter extends JsonConverter<DateTime?, int?> {
   }
 }
 
+/// Top level converter for serializing Nodes map to and from JSON.
+class CanvasesMapConverter extends JsonConverter<
+    Map<String, Map<String, BaseNode>>, Map<String, dynamic>> {
+  /// Creates a new instance of [CanvasesMapConverter].
+  const CanvasesMapConverter();
+
+  @override
+  Map<String, Map<String, BaseNode>> fromJson(Map<String, dynamic> json) =>
+      deserialize(json);
+
+  @override
+  Map<String, dynamic> toJson(Map<String, Map<String, BaseNode>> object) =>
+      serialize(object);
+
+  /// Top level function to deserialize a JSON Map into a map of node ID to node.
+  static Map<String, Map<String, BaseNode>> deserialize(
+          Map<String, dynamic> value) =>
+      {
+        for (final MapEntry(key: key, value: value) in value.entries)
+          key: NodesMapConverter.deserialize(value),
+      };
+
+  /// Top level function to serialize a map of node ID to node into a JSON map.
+  static Map<String, dynamic> serialize(
+          Map<String, Map<String, BaseNode>> canvases) =>
+      {
+        for (final MapEntry(key: key, value: value) in canvases.entries)
+          key: NodesMapConverter.serialize(value),
+      };
+}
+
```

Comment:
This whole thing can be removed.

Reply:
It is still being used. Remember we tried to remove it but then it failed?

Reply:
<img width="487" alt="image" src="https://github.com/Codelessly/CodelesslyAPI/assets/20423471/a9c79894-2adb-4899-891c-1832cc4ed046">